foundry.toml

```diff
@@ -6,8 +6,17 @@ fs_permissions = [
     { access = "read", path = "./lib/dss-test/script/input/"},
     { access = "read", path = "./out/ArbitrumDomain.sol/ArbSysOverride.json"}
 ]
+solc_version = "0.8.16"
+optimizer = false
+# If the optimizer is enabled, optimize for deployment size by using low runs
+optimzier_runs = 1
+via_ir = false
+libraries = [
+    "./lib/dss-exec-lib/src/DssExecLib.sol:DssExecLib:0x0496AbA1f2CD4038cC486993f5CCF97851AE26d3"
+]
 
 [rpc_endpoints]
 mainnet = "${ETH_RPC_URL}"
 
-# See more config options https://github.com/gakonst/foundry/tree/master/config
+[etherscan]
+mainnet = { key = "${ETHERSCAN_API_KEY}" }
```

Comment:
We might need to remove this, because it's breaking CI.
Please check the implications of removing this section.